src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -2088,55 +2087,66 @@
             </attribute>
         </optional>
     </define>
-    
+
     <define name="poem">
-        <!-- Use: poem is a complete poem or fragment thereof. -->
         <a:documentation>  Use: poem is a complete poem or fragment thereof.  </a:documentation>
-        <element name="section">
-            <!-- HTML content model: (a | abbr | address | article | aside | audio | b | bdi | bdo | blockquote | br | button | canvas | cite | code | data | datalist | del | details | dfn | dialog | div | dl | em | embed | fieldset | figure | footer | form | h1 | h2 | h3 | h4 | h5 | h6 | header | hr | i | iframe | img | input | ins | kbd | keygen | label | link | main | map | mark | math | menu | meta | meter | nav | noscript | object | ol | output | p | pre | progress | q | ruby | s | samp | script | section | select | small | span | strong | style | sub | sup | svg | table | template | textarea | time | u | ul | var | video | wbr | (text))* -->
-            <!-- Strict content model: (h1 | span | p | section | img | figure)* -->
-            <ref name="attlist.poem"/> <!-- @epub:type, @class, @id, @title, @xml:space, @xml:lang, @lang, @dir -->
-            <zeroOrMore>
+        <element name="div">
+            <attribute name="class">
+                <value>verse</value>
+            </attribute>
+            <optional>
                 <choice>
                     <ref name="title"/> <!-- h1, h2, h3, h4, h5, h6 -->
```

Comment:
I don't think headers are allowed inside the `<div class="verse">`. Only bridgeheads (previously verse-title) are allowed here, while the headers should be placed outside the `div` as a direct child of the surrounding `<section>` element.

A sorrounding section element with a header is no different than any other types of chapters, so it doesn't need any explicit validation. So it should be enough to just remove the `<ref name="title"/>`. And remove the surrounding `<choice>` since there's only one choice remaining.

Reply:
@josteinaj Sounds good to me!